pyproject.toml

```diff
@@ -20,7 +20,7 @@ classifiers = [
 ]
 dependencies = [
     "numpy<2.0",
-    "astroquery",
+    "astroquery==0.4.7",
     "astropy",
     "astrodbkit>=2.1",
```

Comment:
Perhaps `astrodbkit>=2.1,<2.3` to avoid picking up the 2.3 version which will require astroquery >= 0.4.9?

Reply:
Will do. 

But as soon as the new astrodbkit is released, we'll change this to astroquery >=0.4.8 and astrodbkit >=2.3.